subtitles.go

```diff
@@ -246,6 +246,7 @@ type StyleAttributes struct {
 	WebVTTVertical       string
 	WebVTTViewportAnchor string
 	WebVTTWidth          string
+	WebVTTCSS            string
```

Comment:
Could you rename this attribute to `WebVTTStyle` and order fields alphabetically?

Reply:
Yes, I will